packages/v3/src/components/CvNotification/CvToastNotification.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    role="alert"
+    :kind="kind"
+    :class="[
+      `${carbonPrefix}--toast-notification`,
+      `${carbonPrefix}--toast-notification--${kind}`,
+      { [`${carbonPrefix}--toast-notification--low-contrast`]: lowContrast },
+    ]"
+  >
+    <component
+      :is="icon"
+      :class="`${carbonPrefix}--toast-notification__icon`"
+    />
+    <div :class="`${carbonPrefix}--toast-notification__details`">
+      <h3 :class="`${carbonPrefix}--toast-notification__title`">{{ title }}</h3>
+      <div :class="`${carbonPrefix}--toast-notification__subtitle`">
+        {{ subTitle }}
+      </div>
+      <div :class="`${carbonPrefix}--toast-notification__caption`">
+        {{ caption }}
+      </div>
+    </div>
+    <button
+      type="button"
+      :aria-label="closeAriaLabel"
+      :class="`${carbonPrefix}--toast-notification__close-button`"
+      @click="$emit('close')"
+    >
+      <close20 :class="`${carbonPrefix}--inline-notification__close-icon`" />
+    </button>
+  </div>
+</template>
+
+<script>
+import { computed } from 'vue';
+import { Close20 } from '@carbon/icons-vue';
+import { carbonPrefix } from '../../global/settings';
+import { notificationKinds, notificationIcons } from './consts';
+
+export default {
+  name: 'CvToastNotification',
+  components: { Close20 },
+  props: {
+    /** Type of status */
+    kind: {
+      type: String,
+      default: notificationKinds[0],
+      validator: value => notificationKinds.includes(value),
+    },
+    /** Notification title */
+    title: {
+      type: String,
+      default: '',
+    },
+    /** Notification sub title (supports HTML) */
+    subTitle: {
+      type: String,
+      default: '',
+    },
+    /** Notification caption (supports HTML) */
+    caption: {
+      type: String,
+      default: '',
+    },
+    /** Close button A11y label */
+    closeAriaLabel: {
+      type: String,
+      default: 'Close notification',
+    },
+    /** Use lower contrast version of the notification */
+    lowContrast: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  emits: ['action', 'close'],
```

Comment:
```suggestion
  emits: [
    /**
     * action: emitted on clicking the action button
     */
    'action',
    /**
     * close: emitted one close
     */
    'close',
  ],
```

Reply:
Is there a reason to put the JSDocs in 3 lines instead of 1? Otherwise I'd suggest to go with this:

```javascript
emits: [
  /** Emitted on clicking the action button */
  'action',
  /** Emitted on close */
  'close',
]
```

What do you think?

Reply:
No reason to go with 3 lines, it's just a VSCode habit I've fallen into. 1 Line looks much neater.